colpali_engine/utils/dataset_transformation.py

```diff
@@ -20,6 +20,32 @@ def load_train_set() -> DatasetDict:
     ds_dict = load_dataset(base_path + ds_path)
     return ds_dict
 
+def load_train_set2() -> DatasetDict:
```

Comment:
This function should probably just be in your local machine, no reason to merge it in